src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Logs for AWS Distro for OpenTelemetry Lambda functions
```

Comment:
For title can we use similar approach as 
'Forward Lambda functions Logs with New Relic and OpenTelemetry'